lib/shopify_app/logger.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module ShopifyApp
+  module Logger
+    LOG_LEVELS = { debug: 0, info: 1, warn: 2, error: 3, off: 4 }
```

Comment:
I think it would good to match the Rails level numbers:

https://guides.rubyonrails.org/debugging_rails_applications.html#log-levels

I'm a bit unsure about having 'off' as a level, that feels like overloading this setting. I'm looking into how else we might handle that.

Reply:
Is this not following the Rails level number, just not including `fatal` and `unknown`? Yeah thats just what popped in my head. We could make a separate variable in the config? Or we could just manually check for `:off` in the `enabled_for_log_level?` method?

Reply:
My concern was the `4` here means `off`, but `4` for Rails means 'fatal`. We could map `off` to 6 to avoid any confusion there.